lib/elasticsearch_models/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ElasticsearchModels
-  VERSION = "2.1.0"
+  VERSION = "2.2.0"
```

Comment:
based on our [Gem Development](https://ringrevenue.atlassian.net/wiki/spaces/DEV/pages/750059580/Gem+Development) wiki, we should really be using patch versions during development and only do this "final" version when we are about to release.

E.g., this version should be 
```
2.2.0.pre.1
```

Reply:
Nit :"patch versions" -> "pre-release versions"